keyring_pass/__main__.py

```diff
@@ -0,0 +1,44 @@
+from . import * # idk if this is bad practice, but as a quick fix it works _shrug_
```

Comment:
from what I'm seeing this should be enough:
```suggestion
import sys

import keyring

from keyring_pass import PasswordStoreBackend
```